libs/libcommon/src/libcommon/storage_client.py

```diff
@@ -1,9 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright 2023 The HuggingFace Authors.
 import logging
-from typing import Any, Optional
+from typing import Literal, Optional, Union, cast
 
 import fsspec
+from fsspec.implementations.local import LocalFileSystem
+from s3fs import S3FileSystem  # type: ignore
```

Comment:
it does not support types?

Reply:
No, at least the version we are using.

We should check again once we update fsspec/s3fs.